auth0/README.md

```diff
@@ -0,0 +1,58 @@
+# Agent Check: auth0
+
+## Overview
+
+Use the Datadog-Auth0 integration to view and analyze your log events from Auth0.
+
+## Setup
+
+All configuration happens on the [Auth0 Dashboard][2]. Find details on how to set it up in the [Auth0 Documentation][3]. Datadog automatically creates a dashboard when it detects Auth0 logs.
```

Comment:
We need to create a dashboard it's not create yet. We can remove the line for now.

Reply:
Removed line about Dashboard